frontend/src/api/metrics/getQueryRange.ts

```diff
@@ -18,6 +18,7 @@ export const getMetricsQueryRange = async (
 			error: null,
 			message: response.data.status,
 			payload: response.data,
+			params: props,
```

Comment:
you can use query client to get the props

Reply:
can you give me an example, because I couldn't find it in the documentation